### ISSUE TITLE
Death Compass

### ISSUE THREAD

**Issue by sisby-folk** [B1n-ry/Youre-in-grave-danger#71]
**Is your feature request related to a problem or do you just have an idea that fit the theme of the mod? Please describe.**
As someone who doesn't always use a minimap mod, cracking open f3 to recover items isn't always my vibe. I also just find it hard to navigate by numbers without becoming frustrated.

**Describe the solution you'd like**
I'd love to see a compass that points towards your death point, and that disappears when someone loots the grave (if possible)

**Describe alternatives you've considered**
Not too sure entirely, but i'm open to ideas

**Additional context**
N/A

**Comment by B1n-ry** [comment]
Alright this seems doable. I don't have any idea of a good way of deleting the compass once a grave is claimed, but the rest shouldn't be too hard. I could look at creating an entirely new gui feature with compass overlay and grave selecting, but this would be a bit harder and is thereby not as likely to be implemented. I'll see what I can do though

**Comment by B1n-ry** [comment]
I opted for just a normal compass that will point to your grave, and if you have it in your inventory when you claim your grave it will get deleted (unless you disable it in the config). Grave compasses are by default not given to players, but can be on respawn if you change the config. There's currently no other way of getting a grave compass. The HUD compass felt like a good idea and that it would look cool but in the end I felt like that was too much work and it feels better to keep things simple.

Thank you anyway for the feature request! Let me know if you have any other cool ideas you would like to see come to life in this mod. I just updated the mod on curseforge, modrinth and github releases, and I hope you like the new update!
